app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewState.kt

```diff
@@ -23,7 +24,8 @@ data class ConversationViewState(
     val isFileSharingEnabled: Boolean = true,
     val hasOngoingCall: Boolean = false,
     val hasEstablishedCall: Boolean = false,
-    val lastUnreadMessage : UIMessage? = null
+    val lastUnreadMessage : UIMessage? = null,
+    val securityClassificationType: SecurityClassificationType = SecurityClassificationType.CLASSIFIED
```

Comment:
Accordingly, to documentation the default state should be `NONE`, since this feature is only required for federated environments that have this config enabled =)

Reply:
oupss, that was a leftover of me testing it 🙈  thanks for the catch 🦅